packages/mwp-app-render/src/components/Dom.test.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Dom from './Dom';
+
+const MOCK_BASE_URL = 'https://www.mock-base-url.com';  
+
+const MOCK_PROPS = {
+	appMarkup: '<div>mock app markup</div>',
+	assetPublicPath: '/mock/asset/public/path',
+	baseUrl: MOCK_BASE_URL,
+	head: {
+		title: { toComponent: () => null },
+		meta: { toComponent: () => null },
+		link: { toComponent: () => null },
+		script: { toComponent: () => null },
+	},
+	initialState: {
+		config: {
+			apiUrl: 'https://www.mock-api-url.com',
+			baseUrl: MOCK_BASE_URL,
+			enableServiceWorker: true,
+			requestLanguage: 'fr-FR',
+			supportedLangs: ['en-US', 'de-De', 'es-ES'],
+			initialNow: '1517589465414',
+			variants: 'mock-variants',
+			entryPath: '/ny-tech',
+			media: 'mobile',
+		},
```

Comment:
`initialState` can be an empty object

Reply:
Ah, I did want to have `requestLanguage` in there so we could see the `html` tag in the snapshot. Could I simplify this to be `initialState: { config: requestLanguage: 'fr-FR' } }`?

Reply:
ah right, yes

Reply:
One more question -- do you think it's worth leaving this as-is so `window.APP_RUNTIME` in the snap is more complete?

Reply:
a lot of the `config` spec/content could change over time, but there's no need for this test to try to track all of it unless it has a direct impact on DOM.jsx